flink-autoscaler-standalone/pom.xml

```diff
@@ -163,6 +163,12 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
It's still after the Test related dependencies. You can check `<!-- Test -->` comment, you can move `HikariCP` before it.